### PR TITLE
Add unique wistia player id to have possibility to have multiple players with the same url on one page

### PR DESCRIPTION
**Pull request by MariyaNizovtsova**
Fix the issue #623
Tested on the page with two Wistia players with the same URL and they don't affect each other anymore.

### CODE REVIEW

File: src/players/Wistia.js [hunk 1]
```diff
@@ -24,7 +26,7 @@ export default class Wistia extends Component {
     getSDK(SDK_URL, SDK_GLOBAL).then(() => {
       window._wq = window._wq || []
       window._wq.push({
-        id: this.getID(url),
+        id: this.playerID,
```
Comment (afzaalahmad) [inline src/players/Wistia.js:29]: can we do something like this here:
`id: this.props.id || this.playerID`

File: src/players/Wistia.js [hunk 2]
```diff
@@ -111,7 +113,7 @@ export default class Wistia extends Component {
       height: '100%'
     }
     return (
-      <div key={id} className={className} style={style} />
+      <div id={this.playerID} key={id} className={className} style={style} />
```
Comment (afzaalahmad) [inline src/players/Wistia.js:116]: same goes here:
`id={this.props.id || this.playerID}`

**Comment by CookPete** [comment]
I've added a `config.wistia.playerId` prop (similar to facebook and twitch) to let you pass in your own `playerId` if you want.

**Comment by afzaalahmad** [comment]
@CookPete appreciated :+1:

**Pull request merged**